content/soarian/dstu2/devices/device.md

```diff
@@ -0,0 +1,73 @@
+ 
+# Device
+
+* TOC
+{:toc}
+
+## Overview
+
+For the Device resource query, the API returns a list of charted procedures that include Unique Device Identifiers (UDIs) for the patient.  The list represents the most current procedure with UDI list that has been charted in Soarian.
+
+Note that traditional charting practices have procedures resolved and charted in coding/billing systems and not interfaced back to the EMR, therefore, the procedure record in Soarian may not be complete nor precise to a procedure list provided as part of a patient's billing record.  The list of procedures in completed or aborted state with UDIs will be as charted in Soarian but may not represent a comprehensive state of implanted devices for a variety of operational reasons.  The charting process flow for UDIs, both their implantation and ongoing status, is not fully mature and the charting at the point of care, material management and interventional/surgical systems is evolving.  Clients may continue to chart patient devices outside a UDI process flow and the UDI list will not represent a comprehensive view of patient's devices.  Therefore, the Soarian Clinicals UDI information may not be complete nor representative of current state of a patients implanted devices. 
+Note that the data returned with the UDI may be incomplete or inconsistent because the FDA (Food and Drug Administration) requirement for UDI labelling, effective 2017, states that manufacturers are responsible for populating an FDA database (GUDID) with information related to their devices; this UDI labelling and GUDID population is a new process and inconsistencies and gaps may exist in the data available. Soarian Clinicals has an ability to enter / scan and parse a UDI, query the FDA database (GUDID), and store information returned from the query.  If the GUDID database is not available, empty, or partially populated, Soarian will store the UDI itself and any available information from the GUDID.  The storage format, and therefore API response format is standardized, but GUDID-available data translated and stored may be inconsistent.  The Soarian documentation defines any translation that is performed.  
+The FHIR definition of device is broader than the ONC definition of implantable device.  The API Device response is per the ONC definition of those devices that are implanted and have a UDI.  They are charted per encounter, per procedure where a patient may have
+1 or more visits where each visit may have
+0 or more procedures charted in completed or aborted state where each procedure may have
+0 or more devices with UDIs charted
+
+Ambiguity might exist for intended contexts of date representing a comprehensive state / version of UDIs on a particular date or alternatively UDIs implanted / explanted / serviced on a particular date.  Therefore, to avoid ambiguity and per ONC direction, all UDIs charted in Soarian will be returned for any query and date/date range parameters will not be supported.  
```

Comment:
I don't think this paragraph is needed, there is not date parameter documented below, so it's obvious that it's not supported.